pkg/routeagent_driver/main.go

```diff
@@ -133,18 +147,6 @@ func main() {
 		logger.Fatalf("Error registering the handlers: %s", err.Error())
 	}
 
-	if env.Uninstall {
-		if err := registry.StopHandlers(true); err != nil {
-			logger.Warningf("Error stopping handlers: %v", err)
-		}
-
```

Comment:
Why is this code removed?

Reply:
The stopHandlers is where the actual cleanup happens.

Reply:
Oops!